pkg/conda/meta/parse.go

```diff
@@ -0,0 +1,40 @@
+package meta
+
+import (
+	"encoding/json"
+
+	"golang.org/x/xerrors"
+
+	dio "github.com/aquasecurity/go-dep-parser/pkg/io"
+	"github.com/aquasecurity/go-dep-parser/pkg/types"
+)
+
+type packageJSON struct {
+	Name    string `json:"name"`
+	Version string `json:"version"`
+	License string `json:"license"`
+}
+
+type Parser struct{}
```

Comment:
Could you add a comment about what this parser is for?

Reply:
Added a comment about goal and reference doc for the parser.